docs/examples/example.ipynb

```diff
@@ -505,7 +505,13 @@
     "    }\n",
     "  ]\n",
     "}\n",
-    "```\n"
+    "```\n",
+    "\n",
+    "Note the nomenclature of the above `model` field, `viam:sensor:linux-wifi`. Models are uniquely namespaced as colon-delimited-triplets in the form `namespace:family:name`, and are named according to the Viam API that your model implements. A model with the `viam` namespace is always Viam-provided. Read more about making custom namespaces [here](https://docs.viam.com/extend/modular-resources/key-concepts/#namespace-1).\n",
```

Comment:
```suggestion
    "Note the nomenclature of the above `model` field, `viam:sensor:linux-wifi`. Models are uniquely namespaced as colon-delimited-triplets in the form `namespace:family:name`, and are named according to the Viam API that your model implements. A model with the `viam` namespace is always Viam-provided. Read more about making custom namespaces [here](https://docs.viam.com/extend/modular-resources/key-concepts/#models).\n",
```

We changed the headers a little with soft launch, this is the updated link target for model naming guidance.